Makefile

```diff
@@ -66,6 +66,22 @@ test: clean ## Run tests and generate coverage report
 	${VENV_BIN}/coverage xml
 	${VENV_BIN}/diff-cover coverage.xml --html-report diff-cover.html
 
+testserver:  ## Run an isolated ephemeral instance of Blockstore for use by edx-platform tests
+	docker-compose --project-name blockstore-testserver -f docker-compose-testserver.yml up -d
+	docker exec -t edx.devstack.mysql /bin/bash -c 'mysql -uroot <<< "create database if not exists blockstore_test_db;"'
+	docker exec -t edx.devstack.blockstore-test /bin/bash -c 'source ~/.bashrc && make requirements && make migrate && make testserver-data'
+	# Now run blockstore until the user hits CTRL-C:
+	docker-compose --project-name blockstore-testserver -f docker-compose-testserver.yml exec blockstore /blockstore/venv/bin/python /blockstore/app/manage.py runserver 0.0.0.0:18251
+	# And destroy everything except the virtualenv volume (which we want to reuse to save time):
+	docker-compose --project-name blockstore-testserver -f docker-compose-testserver.yml down
+	docker exec -t edx.devstack.mysql /bin/bash -c 'mysql -uroot <<< "drop database blockstore_test_db;"'
+
+testserver-data:  # Gets run inside the testserver container
+	# Create a staff user so developers can log in and debug issues:
+	echo "from django.contrib.auth import get_user_model; User = get_user_model(); u, _ = User.objects.get_or_create(username='staff', is_staff=True, is_superuser=True); u.set_password('edx'); u.save();" | /blockstore/app/manage.py shell
+	# Configure the API auth header:
+	echo "from django.contrib.auth import get_user_model; User = get_user_model(); u, _ = User.objects.get_or_create(username='edxapp'); from rest_framework.authtoken.models import Token; Token.objects.get_or_create(user=u, key='edxapp-test-key')" | /blockstore/app/manage.py shell
```

Comment:
Nit: Putting Python code in a Python file somewhere and cat/exec'ing it somehow from the file name might make this easier to modify and debug issues later.

Reply:
Yeah, good call. Done in 884b9bf2472f4e6848c6b06d68fdd15cfbc4d8bf